pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1763,6 +1765,19 @@ type BackupScheduleSpec struct {
 	// if MaxBackups and MaxReservedTime are set at the same time, MaxReservedTime is preferred
 	// and MaxBackups is ignored.
 	MaxBackups *int32 `json:"maxBackups,omitempty"`
+	// MaxCompletedBackups specifies the number of completed backups to retain. It should be greater than 0, defaults to 3.
+	// if both MaxCompletedBackups and MaxBackups are set, operator will keep no more than MaxCompletedBackups completed backups,
+	// and no more than MaxBackups backups in total.
+	// +kubebuilder:default:=3
+	// +kubebuilder:validation:Minimum:=1
```

Comment:
For the old backupSchedule, if the user does not configure MaxBackups, which means no clean-up for the backups, with this change, after upgrading the TiDB Operator, only 3 completed backups will be retained, and the others will be cleaned, right? If yes, this is unexpected behavior.